### PR TITLE
Vas0.4.1

### PR DESCRIPTION
**Pull request by adam-intel**

### CODE REVIEW

File: docs_src/phases/phase3.md [hunk 1]
```diff
@@ -60,13 +60,88 @@ which will create the following docker image:
...
+#### RTSP
+
+For an RTSP feed edit the CAMERA0_SRC environment variable to point to the URL of the RTSP feed.
+
+`CAMERA0_SRC=rtsp://{URL}:{PORT}/{STREAM-END-POINT}`
+
+If the RTSP camera has username and password auth append those values to the beginning of the URL. i.e. `CAMERA0_SRC=rtsp://{USERNAME}:{PASSWORD}@{URL}:{PORT}/{STREAM-END-POINT}`.
+
+#### USB camera
+First, add the camera device to the `video-analytic` container.
```
Comment (jim-wang-intel) [inline docs_src/phases/phase3.md:111]:
```suggestion
First, add the camera device to the `video-analytic` service in the docker-compose file.
```

File: docs_src/phases/phase3.md [hunk 2]
```diff
@@ -60,13 +60,88 @@ which will create the following docker image:
...
+devices:
+    - /dev/video0:/dev/video0
+```
+
+Then edit the CAMERA0_SRC environment variable to point to the video source
+`CAMERA0_SRC=/dev/video0`
+
+#### Local video file
+
+To use a local video sample add the path to the video file in the container to the CAMERA0_SRC environment variable.  The keyword `file` lets VAS know the URI is a local file and not a RTSP URL.
```
Comment (jim-wang-intel) [inline docs_src/phases/phase3.md:123]: do we have requirement about what kind of format (eg. mp4, wma, ...etc) for this video file? if so, probably want to list them out.

**Review by jim-wang-intel — COMMENTED** [review]
some text change suggestions only

**Comment by jim-wang-intel** [comment]
overall LGTM

**Review by brian-intel — APPROVED** [review]

**Comment by Walt-Intel** [comment]
Looks good to me.

**Review by jim-wang-intel — APPROVED** [review]

**Pull request merged**